specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.exports.json

```diff
@@ -0,0 +1,799 @@
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ExportExecutionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ExportListResult": {
+      "description": "Result of listing exports. It contains a list of available exports in the scope provided.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of exports.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Export"
+          }
+        }
+      }
+    },
+    "Export": {
+      "description": "An export resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "common-types.json#/definitions/CostManagementProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExportProperties",
+          "title": "Export properties"
+        }
+      }
+    },
+    "ExportProperties": {
+      "description": "The properties of the export.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/CommonExportProperties"
+        }
+      ],
+      "properties": {
+        "schedule": {
+          "description": "Has schedule information for the export.",
+          "$ref": "#/definitions/ExportSchedule"
+        }
+      }
+    },
+    "CommonExportProperties": {
+      "description": "The common properties of the export.",
+      "type": "object",
+      "properties": {
+        "format": {
+          "description": "The format of the export being delivered. Currently only 'Csv' is supported.",
+          "type": "string",
+          "enum": [
+            "Csv"
+          ],
+          "x-ms-enum": {
+            "name": "FormatType",
+            "modelAsString": true
+          }
+        },
+        "deliveryInfo": {
+          "description": "Has delivery information for the export.",
+          "$ref": "#/definitions/ExportDeliveryInfo"
+        },
+        "definition": {
+          "description": "Has the definition for the export.",
+          "$ref": "#/definitions/ExportDefinition"
+        },
+        "runHistory": {
+          "description": "If requested, has the most recent execution history for the export.",
+          "$ref": "#/definitions/ExportExecutionListResult"
+        },
+        "partitionData": {
+          "description": "If set to true, exported data will be partitioned by size and placed in a blob directory together with a manifest file. Note: this option is currently available only for modern commerce scopes.",
```

Comment:
>Note: this option is currently available only for modern commerce scopes. [](http://example.com/codeflow?start=146&length=73)

Does it still make sense to talk about modern here?

Reply:
Removed the term Modern from all specs